toonz/sources/common/tsystem/tfilepath.cpp

```diff
@@ -80,8 +80,42 @@ bool checkForSeqNum(QString type) {
   else
     return false;
 }
+
+bool parseFrame(const std::wstring &str, int &frame, char &letter,
+                int &padding) {
+  if (str.empty()) return false;
+
+  int i = 0, number = 0;
+  while (i < (int)str.size() && str[i] >= L'0' && str[i] <= L'9')
+    number   = number * 10 + str[i++] - L'0';
+  int digits = i;
+  char l =
+      str[i] >= L'a' && str[i] <= L'z'
+          ? str[i++] + ('a' - L'a')
+          : str[i] >= L'A' && str[i] <= L'Z' ? str[i++] + ('A' - L'A') : '\0';
+  if (number == 0 || i < (int)str.size()) return false;
```

Comment:
Please remove or comment out this line.

`  if (number == 0 || i < (int)str.size()) return false;`

This prevents sequenced files starting from 0 to load properly which was fixed in a prior PR